cmd/slirp-hook-sidecar/README.md

```diff
@@ -0,0 +1,25 @@
+# KubeVirt SMBIOS hook sidecar
```

Comment:
Probably best to fill up with slirp related content

Reply:
Done